parser/src/json/compiler.rs

```diff
@@ -607,42 +609,60 @@ impl Compiler {
             let is_required = required.contains(name);
             // Quote (and escape) the name. TODO: probably overkill to use json_dumps here
             let quoted_name = json_dumps(&json!(name));
-            if property_schema == &Value::Bool(false) {
-                if is_required {
-                    bail!("Required property has 'false' schema: {}", name);
+            let property = match self.gen_json(property_schema) {
+                Ok(node) => node,
+                Err(e) => match e.downcast_ref::<UnsatisfiableSchemaError>() {
+                    // If it's not an UnsatisfiableSchemaError, just propagate it normally
```

Comment:
funky!

Reply:
Was just about to comment "funky good or funky bad?" but I suppose you answered that 😆